src/main/java/CommandHandler.java

```diff
@@ -0,0 +1,213 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class CommandHandler {
+    Scanner sc;
+    ArrayList<Task> taskList;
+
+    public CommandHandler() {
+        this.sc = new Scanner(System.in);
+        this.taskList = new ArrayList<>();
+    }
+
+    public void handleCommand() {
+        while (true) {
+            String command = sc.next();
+            switch (command) {
+                case "bye":
+                    handleBye();
+                    return;
+                case "list":
+                    handleList();
+                    break;
+                case "done":
+                    try {
+                        handleDone();
+                    } catch (DukeException e) {
+                        System.out.println("    ____________________________________________________________");
```

Comment:
Maybe you could abstract out this border line as a variable, then call it whenever you need it. Makes the code tidy.